ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane-savage.ts

```diff
@@ -160,19 +227,75 @@ const triggerSet: TriggerSet<Data> = {
       run: (data, matches) => data.suds = matches.effectId,
     },
     {
-      id: 'ASSS Suds Lose',
+      id: 'ASSS Silkie Suds Lose',
+      // Silkie:
       // CE1 Bracing Suds (Wind / Donut)
       // CE2 Chilling Suds (Ice / Cardinal)
       // CE3 Fizzling Suds (Lightning / Intercardinal)
       type: 'LosesEffect',
       netRegex: { effectId: 'CE[1-3]', target: 'Silkie', capture: false },
       run: (data) => delete data.suds,
     },
+    {
+      id: 'ASSS Fresh Puff 1', // 3 puffs in triangle formation
+      type: 'StartsUsing',
+      // use 7774/7775 (Squeaky Clean Left/Right), rather than 7789 (Fresh Puff)
+      // Squeaky Clean will change the effects of two puffs after the Fresh Puff cast
+      // so it is the easiest method to determine mechanic resolution
+      netRegex: { id: '777[45]', source: 'Silkie' },
+      condition: (data) => data.puffCounter === 1,
+      delaySeconds: 9, // delay alert until after Squeaky Clean Left/Right completes to collect Silken Puff effects
+      durationSeconds: 8, // keep alert up until just before Slippery Soap trigger fires
+      alertText: (data, matches, output) => {
+        if (Object.keys(data.silkenPuffs).length !== 3)
+          return output.default!();
+
+        const puffsByLoc: { [location: string]: string } = {};
+        for (const puff of Object.values(data.silkenPuffs)) {
+          if (puff.location !== undefined)
+            puffsByLoc[puff.location] = puff.effect;
+        }
+
+        // See Silken Puff Suds Gain trigger for list of Silken Puff effectIds
+        // By this point, Squeaky Clean Left/Right has changed the N puff and either the SW/SE puff to CE9 (Bracing Suds)
+        // We only care about the unaffected puff's status effect (CEA/CEB) for resolving the mechanic.
+        let stackDir = '';
+        let safeDir = '';
```

Comment:
These always get assigned before use, so I think(?) they don't need an initialization value.

```suggestion
        let stackDir;
        let safeDir;
```